beacon-chain/das/availability_blobs_test.go

```diff
@@ -51,16 +51,20 @@ func Test_commitmentsToCheck(t *testing.T) {
 			name: "commitments within da",
 			block: func(t *testing.T) blocks.ROBlock {
 				d := util.NewBeaconBlockFulu()
-				d.Block.Body.BlobKzgCommitments = commits[:maxBlobs]
 				d.Block.Slot = fulu + 100
+				mb := params.GetNetworkScheduleEntry(slots.ToEpoch(d.Block.Slot)).MaxBlobsPerBlock
```

Comment:
using maxBlobs doesn't work here for this test after adding the blob schedule, need to ask if this is the proper way to fix this test

Reply:
What happened here is that adding the blob schedule means that the value of `maxBlobs` is now the last entry in the BPO rather than just the fulu value (aka the electra max). Yes I think it makes sense for the test cases to look up the max blobs value for the specific period they want to test.